content/guides/output-management.md

```diff
@@ -210,8 +211,8 @@ __webpack.config.js__
 +     new CleanWebpackPlugin(['dist']),
       new HtmlWebpackPlugin({
         title: 'Output Management',
```

Comment:
Technically this should be a diff line too, as a comma is added to the end.
No biggie, just for future ref.